functions/go/fmt/README.md

```diff
@@ -0,0 +1,73 @@
+# fmt
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources
```

Comment:
nit, want to form complet sentences ending with a `.`.

Reply:
Updated.